subiquity/tests/api/test_api.py

```diff
@@ -374,8 +372,9 @@ async def test_v2_add_boot_partition(self):
 
             await inst.post('/storage/v2/reset')
 
-            # these manual steps are expected to be equivalent to just adding
-            # the single partition and getting the automatic boot partition
+            # these manual steps are expected to be mostly equivalent to just
+            # adding the single partition and getting the automatic boot
+            # partition
```

Comment:
It is a bit strange that the boot partition ends up being partition 2. I think we could fix this if we wanted by having v2_add_partition_POST explicitly call add_boot_disk if it needs to, the way partition_disk_handler does. But I guess that may make the gap passed to v2_add_partition_POST no longer valid so blah. Maybe we see if anyone notices :)